### PR TITLE
run maestro with rosa

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: test/e2e/setup/rosa/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+This setup demonstrate how to deploy the Maestro with AWS IoT and AWS RDS PostgreSQL.
+
+1. Run following command to deploy a Maestro server on a given cluster
+
+```sh
+export AWS_IOT_REGION="<your_aws_iot_region>"
+export CLUSTER_REGION="<your_cluster_region>"
+export CLUSTER_VPC="<your_cluster_vpc>"
+export KUBECONFIG="<your_cluster_kubeconfig>"
```
Comment (clyang82) [inline test/e2e/setup/rosa/README.md:13]: No aws credentials provided? use default location?

File: test/e2e/setup/rosa/setup/agent.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+fi
+
+if [ -z "$consumer_id" ]; then
+    echo "consumer id is required"
+    exit 1
+fi
+
+IMAGE_REGISTRY=${IMAGE_REGISTRY:="quay.io/redhat-user-workloads/maestro-rhtap-tenant/maestro"}
+IMAGE_REPOSITORY="maestro"
+IMAGE_TAG=${IMAGE_TAG:-"1de63c6075f2c95c9661d790d164019f60d789f3"}
```
Comment (clyang82) [inline test/e2e/setup/rosa/setup/agent.sh:25]: we should get the commit sha via github api

File: test/e2e/setup/rosa/README.md (hunk 1 above)
Comment (clyang82) [inline test/e2e/setup/rosa/README.md:13]: won't create a cluster?

Comment (clyang82) [reply test/e2e/setup/rosa/README.md:13]: I saw the next section to create a cluster. what is the purpose for this section?

Comment (skeeey) [reply test/e2e/setup/rosa/README.md:13]: This required a logined aws command

File: test/e2e/setup/rosa/setup/agent.sh (hunk 2 above)
Comment (skeeey) [reply test/e2e/setup/rosa/setup/agent.sh:25]: github api? how

File: test/e2e/setup/rosa/README.md (hunk 1 above)
Comment (skeeey) [reply test/e2e/setup/rosa/README.md]: Add Prerequisites part to address this

File: test/e2e/setup/rosa/setup/agent.sh (hunk 2 above)
Comment (skeeey) [reply test/e2e/setup/rosa/setup/agent.sh:27]: this is default value, it can be set outside

File: test/e2e/setup/rosa/README.md (hunk 1 above)
Comment (skeeey) [reply test/e2e/setup/rosa/README.md]: updated the readme

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**